Traceback (most recent call last):
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/utils/import_utils.py", line 1002, in _get_module
    return importlib.import_module("." + module_name, self.__name__)
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/models/__init__.py", line 19, in <module>
    from . import (
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/models/mt5/__init__.py", line 33, in <module>
    from ..t5.tokenization_t5 import T5Tokenizer
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/models/t5/tokenization_t5.py", line 26, in <module>
    from ...tokenization_utils import PreTrainedTokenizer
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/tokenization_utils.py", line 26, in <module>
    from .tokenization_utils_base import (
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/tokenization_utils_base.py", line 73, in <module>
    from tokenizers import AddedToken
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/tokenizers/__init__.py", line 79, in <module>
    from .tokenizers import (
ImportError: libssl.so.10: cannot open shared object file: No such file or directory

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "train_experiment_adapter_from_scratch_mam_config.py", line 13, in <module>
    from utils import *
  File "/raid/cs21mtech11006/Lamol_with_adapter_final/utils.py", line 17, in <module>
    from settings import args, TASK_DICT, SPECIAL_TOKENS, SPECIAL_TOKEN_IDS, FILL_VAL
  File "/raid/cs21mtech11006/Lamol_with_adapter_final/settings.py", line 9, in <module>
    from transformers import OpenAIGPTLMHeadModel, OpenAIGPTTokenizer, OpenAIGPTConfig
  File "<frozen importlib._bootstrap>", line 1032, in _handle_fromlist
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/utils/import_utils.py", line 992, in __getattr__
    module = self._get_module(self._class_to_module[name])
  File "/raid/cs21mtech11006/miniconda3/envs/S-LAmol/lib/python3.7/site-packages/transformers/utils/import_utils.py", line 1007, in _get_module
    ) from e
RuntimeError: Failed to import transformers.models.openai because of the following error (look up to see its traceback):
libssl.so.10: cannot open shared object file: No such file or directory
